README.md

```diff
@@ -186,68 +185,58 @@ Boa3.compile_and_save('path/to/your/file.py', debug=True)
-${path-to-folder}/neo-devpack-dotnet git pull
-${path-to-folder}/neo-devpack-dotnet dotnet build ./src/Neo.TestEngine/Neo.TestEngine.csproj -o {path-to-folder}/Neo.TestEngine
+$ dotnet tool install Neo.Express
+$ dotnet tool install Neo.Test.Runner
 ```
 
 #### Testing
 
-Create a Python Script, import the TestEngine class, and define a function to test your smart contract. In this function
-you'll need to call the method `run()`. Its parameters are the path of the compiled smart contract, the smart
-contract's method, and the arguments if necessary. Then assert your result to see if it's correct.
+Create a Python Script, import the NeoTestRunner class, and define a function to test your smart contract. In this 
+function you'll need to call the method `call_contract()`. Its parameters are the path of the compiled smart contract, 
+the smart contract's method, and the arguments if necessary. Then assert the result of your invoke to see if it's correct.
 
 Your Python Script should look something like this:
 
 ```python
-from boa3_test.tests.test_classes.testengine import TestEngine
-from boa3.internal.neo.smart_contract.VoidType import VoidType
+from boa3_test.test_drive.testrunner.neo_test_runner import NeoTestRunner
 
 
 def test_hello_world_main():
-    root_folder = '{path-to-test-engine-folder}'
+    neoxp_folder = '{path-to-neo-express-directory}'
     project_root_folder = '{path-to-project-root-folder}'
     path = f'{project_root_folder}/boa3_test/examples/hello_world.nef'
-    engine = TestEngine(root_folder)
+    runner = NeoTestRunner(neoxp_folder)
 
-    result = engine.run(path, 'Main')
-    assert result is VoidType
+    invoke = runner.call_contract(path, 'Main')
+    runner.execute()
+
+    assert invoke.result is None
 ```
 
-Alternatively you can change the value of `boa3.env.TEST_ENGINE_DIRECTORY` to the path of your TestEngine:
-> Note: If you intend to run neo3-boa unit tests, this is a requirement
+Alternatively you can change the value of `boa3.env.NEO_EXPRESS_INSTANCE_DIRECTORY` to the path of your .neo-express 
+data file:
 
 ```python
-from boa3_test.tests.test_classes.testengine import TestEngine
+from boa3_test.test_drive.testrunner.neo_test_runner import NeoTestRunner
 from boa3.internal import env
-from boa3.internal.neo.smart_contract.VoidType import VoidType
 
-env.TEST_ENGINE_DIRECTORY = '{path-to-test-engine-folder}'
+env.NEO_EXPRESS_INSTANCE_DIRECTORY = '{path-to-neo-express-directory}'
 
 
 def test_hello_world_main():
     root_folder = '{path-to-project-root-folder}'
     path = f'{root_folder}/boa3_test/examples/hello_world.nef'
-    engine = TestEngine()  # the default path to the TestEngine is the one on env.TEST_ENGINE_DIRECTORY
+    runner = NeoTestRunner()  # the default path to the Neo-Express is the one on env.NEO_EXPRESS_INSTANCE_DIRECTORY
 
-    result = engine.run(path, 'Main')
-    assert result is VoidType
+    invoke = runner.call_contract(path, 'Main')
+    assert invoke.result is None
```

Comment:
this example won't work with Test Runner since it's missing the `execute()` call
```suggestion
    invoke = runner.call_contract(path, 'Main')
    runner.execute()
    assert invoke.result is None
```